redfish-core/lib/pcie.hpp

```diff
@@ -572,6 +573,20 @@ inline void requestRoutesSystemPCIeDevice(App& app)
                             }
                         }
 
+                        if (size_t* property = std::get_if<size_t>(
+                                &pcieDevProperties["LanesInUse"]);
+                            property)
+                        {
+                            if (property == nullptr)
```

Comment:
the property will not be nullptr here because you already check that on 578 lines so this is dead code

Reply:
Good call. Done.